src/StringTagField.php

```diff
@@ -26,6 +26,12 @@
  */
 class StringTagField extends DropdownField
 {
+    private static array $field_validators = [
+        // Disable validation as StringTagField is a special case where labels can be the values
+        // rather than the keys always being the values - for instance when creating new tags
+        OptionFieldValidator::class => null,
```

Comment:
This means this field now has _no_ validation, where it used to have validation through inheritance. That's no good.

Same with TagField.